wlauto/common/linux/device.py

```diff
@@ -137,6 +137,10 @@ def abi(self):
         return self._abi
 
     @property
+    def supported_eabi(self):
```

Comment:
Yes, it should be changed. This is a WA `Device` property -- it is not android specific (it should be implemented for both Android and Linux devices). It just so happens that for Android it is implemented by querying the similarly-named Android property, but that is an implementation detail.